Sources/Publish/API/PlotComponents.swift

```diff
@@ -61,11 +63,17 @@ public extension Node where Context == HTML.DocumentContext {
             .unwrap(location.imagePath ?? site.imagePath, { path in
                 let url = site.url(for: path)
                 return .socialImageLink(url)
-            })
+            }),
+            .forEach(additionalWholeSiteHeadNodes + additionalNodes){$0}
         )
     }
 }
 
+public extension Node where Context == HTML.DocumentContext {
+    /// Additional nodes of head. These nodes will be added to head of whole site.
+    static var additionalWholeSiteHeadNodes: [Node<HTML.HeadContext>] = []
```

Comment:
I don't think adding globally mutable state is a good idea. I understand the use case for this, but I think it should be solved in other ways, for example by creating a site-specific `head` extension which then composes these universal nodes into Publish's `head(for...)` call.

Reply:
I agree with you. Global variable is not a clean way to do.

Let me consider a better implementation.

Reply:
I'd just remove it for now, so that we can merge the other change in 🙂

Reply:
I have removed it.